app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
+    if files_names[1] == files_names[2]:
```

Comment:
You should additionally check that the `cp` word is present in a command and it is placed in the first place. Additionally, there is a need to check that the input command contains exactly three elements according [to the checklist](https://github.com/AnyoneClown/py-copy-file/blob/develop/checklist.md#code-efficiency). Also, don't use indexes to retrieve elements from the split command, prefer unpacking and writing values to variables after the length check.